python/GafferDispatchUI/DispatcherUI.py

```diff
@@ -131,6 +131,7 @@
 	GafferDispatch.TaskNode,
 
 	"layout:customWidget:dispatchButton:widgetType", "GafferDispatchUI.DispatcherUI._DispatchButton",
+	"layout:activator:sequence", lambda node : node.requiresSequenceExecution(),
```

Comment:
Prefer `node["task"].requiresSequenceExecution()`. It's a minor distinction, but the general idea is that plugs provide the public interface to nodes, and that we might protect the methods on the nodes more heavily in future (they're already protected in C++, but not in Python).

More importantly though, this just doesn't work, and has hidden every `batchSize` widget unconditionally. There are multiple problems :

- The activator needs to be defined on the parent being displayed by the PlugLayout. That's the `dispatcher` plug in this case, not the node.
- The return value is the opposite of what we need.
- There's a related bug in PlugLayout that meant that even when the above is sorted, the result doesn't show up when switching between the "Advanced" and "Dispatcher" tabs on a PythonCommand. I've fixed that in https://github.com/GafferHQ/gaffer/pull/4193.

Here's a commit that sorts the rest out : https://github.com/johnhaddon/gaffer/commit/11b5934ca85b032d75e53acc2f115bf005510793.

Reply:
Addressed in 0218368